### ISSUE TITLE
workbench - Duplicate Case Number

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/CBRADMS#99]
from user feedback:
>Can duplicate case number be a filter option and be displayed on the workbench? Or could there be another way to see all cases related to the duplicate case?

**Comment by HansVraga** [comment]
I thought we could do this through use of tags, or planned to do this through use of tags. @mitch-wim if you can verify that, you can close this - otherwise ping me and we'll take a look together.

**Comment by aaronstephenson** [comment]
I implemented the workbench filter. It should list all cases that are duplicates of the selected case (but not the selected case itself). They are free to use tags if they wish, that would also work.

**Comment by aaronstephenson** [comment]
from user feedback:
> The Duplicate drop-down list has been added to Filter Cases as an option, but the duplicate case numbers are not displayed on the Workbench. Please add “of” after “Duplicate” on the filter options to match the Case Records page. When you filter using the Duplicate filter option, it does not bring up the original case only the cases that are duplicates of the original. Is it possible to have the original case pull up along with the duplicates, or can we make a case a duplicate of itself so that it shows up on the filtered workbench?

**Comment by aaronstephenson** [comment]
After rereading this a few times, it finally occurs to me that they're asking for the duplicate case numbers to appear in the case list table as a field in each row, not just as a filter field.

**Comment by aaronstephenson** [comment]
I have added the "Duplicate of" field to the case list table.

I have also implemented an altered query response where searching for a duplicate will return both the records that are the duplicate of the case ID submitted, _and_ the original case. To that end, I have decided to change the filter field name to just "Duplicate", since we are no longer searching for cases that are just duplicates, but cases that are duplicates and the original.
